theme_anelusia/views/snippets/s_key_images.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_key_images" inherit_id="website.s_key_images">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc5" separator=" "/>
+    </xpath>
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_key_images_default_image_1/web_editor/geometric/geo_square_6.svg</attribute>
+        <attribute name="data-shape">web_editor/geometric/geo_square_6</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">s_key_images_default_image_1.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
```

Comment:
This attribute doesn't seem to be needed, could you check ?

Other occurrences as well

Reply:
I think it's still necessary, but I'm not 100% sure 🤔. 
Let's see during the tech. review what would be the best solution :) .